README.md

```diff
@@ -46,14 +46,15 @@ We are a friendly community.
 # Documentation
 * https://freshrss.github.io/FreshRSS/en/
 
-# Installation
+# [Installation](https://freshrss.github.io/FreshRSS/en/users/01_Installation.html)
 1. Get FreshRSS with git or [by downloading the archive](https://github.com/FreshRSS/FreshRSS/archive/master.zip)
 2. Dump the application on your server (expose only the `./p/` folder)
 3. Add write access on `./data/` folder to the webserver user
 4. Access FreshRSS with your browser and follow the installation process
 	* or use the [Command-Line Interface](./cli/README.md)
 5. Everything should be working :) If you encounter any problem, feel free [contact us](https://github.com/FreshRSS/FreshRSS/issues).
 6. Advanced configuration settings can be seen in [config.default.php](./config.default.php) and modified in `data/config.php`.
+7. When using Apache, enable [`AllowEncodedSlashes`](http://httpd.apache.org/docs/trunk/mod/core.html#allowencodedslashes) for additional compatibility with mobile clients.
```

Comment:
The French "better compatibility" sounds clearer.

Reply:
Fixed :-)